org.videolan.VLC.Plugin.fdkaac.json

```diff
@@ -0,0 +1,68 @@
+{
+  "id": "org.videolan.VLC.Plugin.fdkaac",
+  "build-extension": true,
+  "runtime": "org.videolan.VLC//stable",
+  "sdk": "org.freedesktop.Sdk//1.6",
+  "tags": ["free"],
```

Comment:
Remove this, nothing else uses it.

Reply:
you mean the tag or the //version in runtime and sdk ?

Reply:
tag.